src/components/Asset/AssetActions/index.tsx

```diff
@@ -65,14 +65,11 @@ export default function AssetActions({
       setFileIsLoading(true)
 
       const asset = values?.services?.[0].files?.[0].url || ddo.id
-      const providerUrl =
```

Comment:
the above comment was also the reason for this here, so as that the file info in the publish preview works